examples/bbengfort/testing.ipynb

```diff
@@ -0,0 +1,64 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Visual Diagnosis of Text Classification with Baleen \n",
+    "\n",
+    "This notebook has been created as part of the [Yellowbrick user study](http://www.scikit-yb.org/en/latest/evaluation.html). I hope to explore how visual methods might improve the workflow of text classification on a small to medium sized corpus. \n",
+    "\n",
+    "## Dataset \n",
+    "\n",
+    "The dataset used in this study is a sample of the [Baleen Corpus](http://baleen.districtdatalabs.com/). The Baleen corpus has been ingesting RSS feeds on the hour from a variety of topical feeds since March 2016, including news, hobbies, and political documents and currently has over 1.2M posts from 373 feeds. [Baleen](https://github.com/bbengfort/baleen) (an open source system) has a sister library called [Minke](https://github.com/bbengfort/minke) that provides multiprocessing support for dealing with Gigabytes worth of text. \n",
+    "\n",
+    "The dataset I'll use in this study is a sample of the larger data set that contains 68,052 or roughly 6% of the total corpus. For this test, I've chosen to use the preprocessed corpus, which means I won't have to do any tokenization, but can still apply normalization techniques. The corpus is described as follows:\n",
+    "\n",
+    "Baleen corpus contains 68,052 files in 12 categories.\n",
+    "Structured as:\n",
+    "\n",
+    "- 1,200,378 paragraphs (17.639 mean paragraphs per file)\n",
+    "- 2,058,635 sentences (1.715 mean sentences per paragraph).\n",
+    "\n",
```

Comment:
It may be worth it; I was just adding the stub, you can see that I went with more simple data loading rather than using a CorpusReader object, which is what spit out the description you saw above. I have some text frequency visualizers in there now, which also show similar information, but with yellowbrick!